docs/en_US/install/wiiu.md

```diff
@@ -87,6 +92,21 @@ After installing Pretendo, you must register a Pretendo Network ID (PNID). There
 	A Pretendo Network ID may not use the same username as an account already linked to your Wii U! If you have any existing Nintendo Network IDs on your Wii U which share the username you wish to use, those accounts MUST be removed from your console first.
 </div>
 
+## After you set up your Pretendo Network ID on your Wii U.
+Shut off your console, and put the SD Card into your machine.
+
+Place the downloaded `30_nimble.rpx` file on your SD card at `sd:/wiiu/environments/aroma/modules/setup`. 
+
+<img src="/assets/images/docs/install/wiiu/pnid/nimbus-sd.png" width="100%">
+
+Turn on your console.
+
+After your console has booted up, press the button combo `L + Down + SELECT` on the Wii U GamePad to open the Aroma plugin menu.
+
+<img src="/assets/images/docs/install/wiiu/pnid/aroma-plugins-list.jpg" width="100%">
+
+Choose the plugin 'Inkay', then click A on 'BOSS settings' inside of Inkay, then click A on 'Unregister Wara Wara Plaza BOSS tasks' after that restart your console fully.
+
```

Comment:
This would probably be better placed before PNID setup. It feels odd putting this additional patch guide in the middle of the account setup steps, especially since it doesn't really have much to do with the account

Reply:
Thanks for the advice, will change up a few things soon when I have time. Very much appreciated.

Reply:
I changed it so its a part of the installation process do you think that's good or should I still make it its own separate thing?